client_options.go

```diff
@@ -94,7 +102,7 @@ func WithOnReconnect(handler OnReconnectHandler) ClientOption {
 // Default values for hostname is "127.0.0.1" and for port is 1883.
 func WithTCPAddress(host string, port uint16) ClientOption {
```

Comment:
We can add another `Option` named `WithAddress` and mark this one as `DEPRECATED` and add a comment to use the new `Option`.

Reply:
Fixed 🙇